libs/s25main/network/GameClient.cpp

```diff
@@ -1695,7 +1710,7 @@ void GameClient::SetPause(bool pause)
             return;
         framesinfo.isPaused = true;
         framesinfo.frameTime = FramesInfo::milliseconds32_t::zero();
-    } else if(replayMode)
+    } else if(replayMode || aiBattleMode_)
```

Comment:
Similar to above: The server needs to be involved.

Reply:
Pause seems to work either way. I removed this change.